includes/subscriptions/templates/html-wcpay-deactivate-warning.php

```diff
@@ -20,8 +20,7 @@
 					<p>
 						<?php
 							printf(
-								// Translators: $1 $2 $3 are placeholders are opening and closing HTML link tags, linking to documentation.
-								// Translators: $4 $5 placeholders are opening and closing strong HTML tags.
+								// translators: $1 $2 $3 are placeholders are opening and closing HTML link tags, linking to documentation. $4 $5 placeholders are opening and closing strong HTML tags.
```

Comment:
```suggestion
								// translators: $1 $2 $3 placeholders are opening and closing HTML link tags, linking to documentation. $4 $5 placeholders are opening and closing strong HTML tags.
```
Suggestion: remove extraneous `are`

Reply:
ty ty!